gradio/routes.py

```diff
@@ -699,7 +699,12 @@ def component_server(body: ComponentServerBody):
                 block = state[component_id]
             else:
                 block = app.get_blocks().blocks[component_id]
-            fn = getattr(block, body.fn_name)
+            fn = getattr(block, body.fn_name, None)
+            if fn is None or not getattr(fn, "_is_server_fn", False):
```

Comment:
`None` wouldn't have the attribute either, so this simplifies to
```suggestion
            if not getattr(fn, "_is_server_fn", False):
```

Reply:
makes sense, thanks!

Reply:
actually needed for type check :/